moreinterp/coreutils/coreutils.go

```diff
@@ -1 +1,70 @@
+// Package coreutils provides a middleware for the interpreter that handles
+// core utils commands like cat, chmod, cp, find, ls, mkdir, mv, rm, touch
+// and xargs.
+//
+// This is particularly useful to keep the max compability on Windows where
+// these core utils are not available, unless when installed manually by the
+// user.
 package coreutils
+
+import (
+	"context"
+
+	"github.com/u-root/u-root/pkg/core"
+	"github.com/u-root/u-root/pkg/core/cat"
+	"github.com/u-root/u-root/pkg/core/chmod"
+	"github.com/u-root/u-root/pkg/core/cp"
+	"github.com/u-root/u-root/pkg/core/find"
+	"github.com/u-root/u-root/pkg/core/ls"
+	"github.com/u-root/u-root/pkg/core/mkdir"
+	"github.com/u-root/u-root/pkg/core/mv"
+	"github.com/u-root/u-root/pkg/core/rm"
+	"github.com/u-root/u-root/pkg/core/touch"
+	"github.com/u-root/u-root/pkg/core/xargs"
+	"mvdan.cc/sh/v3/interp"
+)
+
+var coreUtils = map[string]func() core.Command{
```

Comment:
perhaps call this `commands`, given that we're already in a package called coreutils. or even better, to be clearer that these are constructors, something like `commandBuilders`.